DESCRIPTION

```diff
@@ -52,13 +52,16 @@ Depends:
 Imports:
     bayestestR (>= 0.13.0),
     effectsize (> 0.8.2),
-    insight (>= 0.18.8),
+    insight (>= 0.19.1),
     parameters (>= 0.20.0),
     performance (>= 0.9.2),
-    datawizard (>= 0.6.5),
+    datawizard (>= 0.7.0),
     stats,
     tools,
     utils
+Remotes:
+    easystats/effectsize,
+    easystats/datawizard
```

Comment:
So once I submit report I will have to remove this, and then report will fail until you update datawizard, but since you will do it at the same thing, everything will be fine, right?

Reply:
Because it seems that the CRAN version is still 0.6.5. So did you submit the new version? I'm ready to submit now but don't know if I should do it right away or wait for something on your end?

Reply:
Alright. I'll submit report ASAP